### ISSUE TITLE
Provide feedback when bulk upload fails.

### ISSUE THREAD

**Issue by philipmjohnson** [radgrad/radgrad#271]
Please do work for this task in a branch called issue-271.

If the bulk upload fails for some reason, the only way to know is to look in Chrome DevTools at the console output.  

It would be better to put the output (including errors) into the UI somehow.

**Comment by cammoore** [comment]
This was my bad. Changed code from CourseInstances.find().fetch() to CourseInstances.findDoc(). findDoc throws Meteor error when there are no matches. find().fetch() returns an empty array.

**Comment by cammoore** [comment]
The bug was on the client side. We don't create users on the client only on the server. When the client tried to run the Meteor method it didn't create the user then when it tried to create the feed for the new user that would throw an Error. I wrapped the bulkUpload method call in if (Meteor.isServer). Now, we don't get the client side errors.
Added stack traces to all the new Meteor.Error calls. This should give us a better idea what is going wrong.
